### PR TITLE
Adding .nvmrc so we can use nvm use instead of hunting for version

### PR DESCRIPTION
**Pull request by zoobot**
We should move this repo to node v16 at least!
Change-type: minor

Signed-off-by: zoobot <rommims@gmail.com>

Resolves: #   
Change-type: major|minor|patch   
Depends-on: <url>   
See: <url>   

---
Please check the CONTRIBUTING.md file for relevant information and some
guidance. Keep in mind that the CLI is a cross-platform application that runs
on Windows, macOS and Linux. Tests will be automatically run by balena CI on
all three operating systems, but this will only help if you have added test
code that exercises the modified or added feature code.

Note that each commit message (currently only the first line) will be
automatically copied to the CHANGELOG.md file, so try writing it in a way
that describes the feature or fix for CLI users.

If there isn't a linked issue or if the linked issue doesn't quite match the
PR, please add a PR description to explain its purpose or the features that it
implements. Adding PR comments to blocks of code that aren't self explanatory
usually helps with the review process.

If the PR introduces security considerations or affects the development, build
or release process, please be sure to highlight this in the PR description.

Thank you very much for your contribution!

### CODE REVIEW

**Comment by ab77** [comment]
@zoobot Change-type needs to be the last line.. I believe, otherwise no-bueno. Also, if the tests pass - just self-certify it, otherwise it will never get merged. Especially since this is cosmetical.

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+12.22.12
```
Comment (thgreasi) [inline .nvmrc:1]: How about just 12, so that it gets the latest version rather than a pinned one?
```suggestion
12
```

**Review by thgreasi — COMMENTED** [review]
Let's also change this to be `Change-type: patch` since the convention of using `minor` in end-user facing changelogs is for when the PR actually adds any new feature for them.

**Comment by thgreasi** [comment]
@balena-ci rebase

**Review by thgreasi — APPROVED** [review]

**Comment by thgreasi** [comment]
@zoobot can you please rebase over master so that we avoid the extra merge commit and can keep the history linear?
image

**Comment by ab77** [comment]
@zoobot LGTM

**Comment by ab77** [comment]
> @balena-ci rebase

@thgreasi @zoobot I suspect versionbot rebases are, under certain circumstances, won't work. Suggest you use built in functionality in Git UI from the drop down:

image

**Pull request merged**